src/util.js

```diff
@@ -39,22 +39,22 @@ function varlinkCallChannel(channel, method, parameters) {
                 reject(msg);
             } else if (json.parameters) {
                 // debugging
-                resolve(json.parameters)
+                resolve(json.parameters);
             } else
-                reject("protocol error: reply has neither parameters nor error: " + reply);
+                reject(new Error("protocol error: reply has neither parameters nor error: " + reply));
         }
 
         channel.addEventListener("close", on_close);
         channel.addEventListener("message", on_message);
         channel.send(encoder.encode(JSON.stringify({ method, parameters: (parameters || {}) })));
         channel.send([0]); // message separator
-    })
+    });
 }
 
 function varlinkCallError(error) {
     let str = "";
-    error.error ? str += error.error.toString() : str;
-    error.parameters.reason ? str += " " + error.parameters.reason.toString() : str;
+    str = error.error ? str + error.error.toString() : str;
+    str = error.parameters ? str + " " + error.parameters.reason.toString() : str;
```

Comment:
I fixed that crash independently in PR #13, as it was making tests fail (and the page Oopsing) on master. So you can take this out on your next rebase.